src/events/message/messageDelete.ts

```diff
@@ -33,12 +33,12 @@ export default class extends Event {
 						content: message.content ? `**C:** ${Util.cleanContent(message.content, message.channel)}\n` : "",
 						reply: message.type === "REPLY" ? `**Reply to:** https://discord.com/channels/${message.reference?.guildId}/${message.reference?.channelId}/${message.reference?.messageId}\n` : "",
 						sticker: message.stickers.first() ? `**S:** ID: \`${message.stickers.first()?.id}\` | **Name:** ${message.stickers.first()?.name} | **Format:** ${message.stickers.first()?.format}\n` : "",
-						attachment: message.attachments.first() ? `**A**: ${message.attachments.first()?.proxyURL}\n` : "",
+						attachment: message.attachments.first() ? `**A**: ${message.attachments.map((attach: any) => `**${attach.name}**\n${attach.proxyURL}`).join("\n")}\n` : "",
```

Comment:
```suggestion
						attachment: message.attachments.first() ? `**A**: ${message.attachments.map((attach) => `**${attach.name}**\n${attach.proxyURL}`).join("\n")}\n` : "",
```
Why is `attach` typed as `any`? It can only be `MessageAttachment`, which TS can infer on its own too, so I'm not sure why we'd override that.